ui/app/src/state/epics/content.ts

```diff
@@ -283,12 +286,14 @@ const content: CrafterCMSEpic[] = [
           map(({ item: path }) => {
             const editableAsset = isEditableAsset(payload.path);
             if (editableAsset) {
+              const extension = getFileExtensionFromPath(path);
               return showCodeEditorDialog({
                 authoringBase: state.env.authoringBase,
                 site: state.sites.active,
                 path,
-                type: 'asset',
-                onSuccess: payload.onSuccess
+                mode: getEditorModeFromExtension(extension),
```

Comment:
@rart this is not directly related with the original issue, I need to discuss this changes with you